backend/tests/userUpdateTest.test.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-undef */
+import mongoose from 'mongoose';
+import supertest from 'supertest';
+import app from '../app';
+import User from '../models/User';
+
+import userUtils from './testUtils/userTestUtils';
+import updateUtils from './testUtils/updateUserUtils';
+
+const request = supertest(app);
+
+let user;
+let token;
+
+beforeAll(async () => {
+  await User.deleteOne({
+    username: 'leon',
+  });
+
+  await request.post('/api/users/').send(userUtils.newValidUser);
+
+  user = await updateUtils.userInDb();
+});
+
+describe('Login User API Tests', () => {
```

Comment:
Login should be update?

Reply:
Forgot to change the test description, will change that.